### ISSUE TITLE
userSchema roles 

### ISSUE THREAD

**Issue by kyahAndoy3zero** [jonasschmedtmann/complete-node-bootcamp#182]
<img width="432" alt="image" src="https://user-images.githubusercontent.com/87178979/215499596-cb6387a5-fe26-444a-80a2-de983a6a9a18.png">

When a new user sign up and specified that the role is admin. It will create a new user but the role is still user instead of admin.

**Comment by ayushkavad** [comment]
I have faced the same problem as yours when i was creating a new user. it will create a new user and set role as default.
solution is that you need to specified the role into signup controller  

![Screenshot from 2023-02-03 11-30-46](https://user-images.githubusercontent.com/69857046/216526267-c496d6d7-4fb9-4680-853a-889b510d9234.png)

**Comment by ayushkavad** [comment]
I have faced the same problem as yours when i was creating a new user. it will create a new user and set the role as default.
solution is that you need to specified the role into signup controller.

![Uploading Screenshot from 2023-02-03 11-30-46.png…]()

**Comment by ayushkavad** [comment]
I have faced the same problem as yours when i was creating a new user. it will create a new user and set the role as default.
solution is that you need to specified the role into signup controller.

![Screenshot from 2023-02-03 11-30-46](https://user-images.githubusercontent.com/69857046/216527689-5cf059e1-6951-42ee-bf1f-6080abb091e2.png)

**Comment by kyahAndoy3zero** [comment]
Yeah I did that already. I guess there is not much difference since it doesn't have a required field in the schema.
